src/components/templates/ScreeningRoom/useScreeningRoom.ts

```diff
@@ -119,24 +119,21 @@ export const useScreeningRoom = (screeningRoomVenueId: string) => {
   );
 
   const searchedVideos = useMemo(() => {
-    if (!searchQuery)
-      return filteredVideosBySubCategory.slice(0, displayedVideosAmount);
+    if (!searchQuery) return filteredVideosBySubCategory;
 
     return fuseVideos
       .search(searchQuery)
-      .slice(0, displayedVideosAmount)
       .map((fuseSearchItem) => fuseSearchItem.item);
-  }, [
-    searchQuery,
-    fuseVideos,
-    filteredVideosBySubCategory,
-    displayedVideosAmount,
-  ]);
+  }, [searchQuery, fuseVideos, filteredVideosBySubCategory]);
+
+  const displayedVideos = searchedVideos
+    .sort((a, b) => a.title.localeCompare(b.title))
+    .slice(0, displayedVideosAmount);
 
-  const hasHiddenVideos = searchedVideos.length > displayedVideosAmount;
+  const hasHiddenVideos = searchedVideos.length > displayedVideos.length;
```

Comment:
and can you explain why was that changed?
I guess `displayedVideosAmount` should have the same value. 🤔

Reply:
Since `displayedVideos` created from a `.slice(0, displayedVideosAmount)` it should at most have an array length of `displayedVideosAmount`; but I expect it could possibly have less elements than that if `displayedVideosAmount` is set to a value higher than the total length of `searchedVideos`

Reply:
@sunny-viktoryia that logic was broken. Now it's fixed